Adapt poster

alwil opened the pull request:
Hi Bjorn, 

I've added this small helper function to create posters for sessions automatically. Once included, It can be incorporated in the website together with the calendar creator

alwil committed:
first attempt to programatically create poster for next R session

alwil committed:
generalize poster creation

alwil committed:
added outfile path as argument

bbartholdy committed:
fix missing template_path

bbartholdy commented:
It wasn't working for me. I was getting an error

```r
Error in `magick::image_read()`:
! path must be URL, filename or raw vector
Hide Traceback
    ▆
 1. └─tudrcafe:::update_poster(sched[1, ])
 2.   └─magick::image_read(template_path) at [tudrcafe/R/utils.R:79:3](vscode-file://vscode-app/usr/share/positron/resources/app/out/vs/code/electron-sandbox/workbench/workbench.html#)
```

`template_path <- system.file("poster_template.png", package = 'tudrcafe', out_path = "")` was returning an empty string so I moved `out_path` as a function argument instead as a possible solution, but maybe see if it works for you.

alwil committed:
adapting path

alwil commented:
oh man, I was adding `out_path` as a last minute thing. it was *supposed to be* function argument, just added it one line below. 

`"."` didn't work for me. changed it to `""`. now it just creates a file in the root. Does it work for you?
